micros/auth/services/auth-service.js

```diff
@@ -43,6 +45,30 @@ exports.recaptchaV3 = async function (response_key) {
 };
 
 exports.createUser = async function (reqfullName, reqEmail, hashPassword) {
+  const postData = {};
+  postData.id = "1";
+  postData.fullName = reqfullName;
+  postData.email = reqEmail;
+  postData.password = hashPassword;
+  postData.userName = reqEmail;
+  let gen_hmac = hmac.sign(JSON.stringify(postData), appConfig.HMAC_SECRET_KEY);
+  let axiosConfig = {
+    headers: {
+      "Content-Type": "application/json;charset=UTF-8",
+      "X-Cloud-Signature": `${gen_hmac.toString()}`,
```

Comment:
Please use `process.env.HMAC_HEADER_NAME` instead of hardcoded `X-Cloud-Signature`